php/class-fieldmanager-field.php

```diff
@@ -390,15 +455,17 @@ public function set_options( $label, $options ) {
 	/**
 	 * Generates all markup needed for all form elements in this field.
 	 * Could be called directly by a plugin or theme.
-	 * @param array $values the current values of this element, in a tree structure if the element has children.
-	 * @return string HTML for all form elements.
+	 *
+	 * @param array $values The current values of this element, in a tree structure
+	 *                      if the element has children.
+	 * @return string       HTML for all form elements.
 	 */
 	public function element_markup( $values = array() ) {
 		$values = $this->preload_alter_values( $values );
-		if ( $this->limit != 1 ) {
+		if ( 1 != $this->limit ) {
```

Comment:
Not `!==`? (Applies in a few places.)

Reply:
I started making that change yesterday but decided that in order to do so, we need to take further precautions about type. I think that technically someone could use `'limit' => '1'` now and everything would still work. I logged #646 for that change.